sklift/models/models.py

```diff
@@ -97,7 +97,13 @@ def fit(self, X, y, treatment, estimator_fit_params=None):
         treatment_values = np.unique(treatment)
 
         if len(treatment_values) != 2:
-            raise ValueError("Expected only two unique values, got %s" % len(treatment_values))
+            raise ValueError("Expected only two unique values in treatment vector, got %s" % len(treatment_values))
+
+        check_is_binary(y)
+        target_values = np.unique(y)
+
+        if len(target_values) != 2:
+            raise ValueError("Expected only two unique values in target vector, got %s" % len(target_values))
```

Comment:
The model `SoloModel` can work with a continuous target. 
But metrics for this type of target are not implemented at the moment.